runner/requirements.live-ai.txt

```diff
@@ -6,7 +6,7 @@ Pillow==10.3.0
 python-multipart==0.0.9
 uvicorn==0.34.0
 huggingface_hub==0.23.2
-triton>=2.1.0
+triton>=3.1.0
```

Comment:
Oof, I remember having a hard time because fo this `triton` dep version. Make sure nothing breaks (I recall maybe it was catastrophic like it didn't even startup. so if you can get a stream going it should be fine)

Reply:
I noticed there was a difference in versions. I've reverted it back to triton>=2.1.0, but livepeer/comfyui-base will come with `Version: 3.2.0`. Ideally would like to minimize the python package installs here to minimize risk of breaking a custom node

Reply:
Does it work on triton 3.2.0? If it dos, feel free to remove it from here. This file is only installed on the comfyui image.